src/modules/commander/PreflightCheck.cpp

```diff
@@ -142,7 +142,7 @@ static bool magnometerCheck(orb_advert_t *mavlink_log_pub, unsigned instance, bo
 
 	if (ret) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: MAG #%u UNCALIBRATED", instance);
+			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: MAGNETOMETER #%u UNCALIBRATED.", instance);
```

Comment:
text already improved in this more recent PR:  https://github.com/PX4/Firmware/pull/10586/files#diff-a2bcb238e25d8a97ab2aee48f54dbc48R126